Essentials/src/main/java/com/earth2me/essentials/Settings.java

```diff
@@ -1948,4 +1948,10 @@ public boolean showZeroBaltop() {
     public int getMaxItemLore() {
         return config.getInt("max-itemlore-lines", 10);
     }
+
+    @Override
+    public boolean isPayModifierEnabled() {
+        return config.getBoolean("pay-modifier-enabled", true);
```

Comment:
Should be false here as well if we want it to default to false if a new config isn't being generated